client/style.css

```diff
@@ -0,0 +1,37 @@
+h1 {
+	color: orange;
+}
+.nav-bar {
+	padding-top: 20px;
+	max-width: 1600px;
+}
+.footer {
+	padding-top: 20px;
+	max-width: 1600px;
+}
+#wrapper {
+	padding-left: 100px;
+	padding-right: 80px;
+	width: 1600px;
+	display: grid;
+	grid-template-columns: 450px 450px 400px;
+	gap: 25px;
+	grid-auto-rows: minmax(1000px, auto);
```

Comment:
If not all of them to relative sizing, please just this one to prevent horizontal scrolling